### DISCUSSION TITLE
How to run FLux and SD 3.5 in SDNext?

### DISCUSSION THREAD

**Discussion by TanvirHafiz** [vladmandic/sdnext#3523] [General]
I installed and tried to download SD 3.5 model from Civitai from inside the webui. then manually downloaded and posted in the models directory. but it wouldnt generate image and gives error that it is not supported. how can i run flux (with lora support) and SD 3.5 in SDNext?

**Comment by vladmandic** [comment]
start with networks -> models -> reference and select a model.
99% of flux and sd3 downloads on civitai are not a full model, they are only single model component (unet).

**Reply by panton41** [reply]
Then how are you supposed to use them?

Because I have program on Apple Silicon called Draw Things that "Just Works" by downloading those single model components onto my hard drive and telling it to import the file and I was very slowly churning out pictures all night long, but SD.Next (on my much faster Linux computer) doesn't have a clue what to do with them.

I don't want the "doing this works for me" that you keep saying because it doesn't. I want actual step-by-step directions because the official documention is beyond useless and saying NOTHING like what you just said there. (Which, again, doesn't actually work.)

I followed those directions and get the same error with files downloaded from the UI as I do just downloading them in my browser and dragging and dropping, so, no, I severely doubt thousands of people are doing it successfully. Honestly, Flux support seems completely broken.

**Reply by vladmandic** [reply]
in general, replying to a 3 month old closed thread is almost never a way to go, but anyhow...
first, tone it down a bit - this is open source and fueled by voluteer work.
second, not sure what is not clear - load a model and then load a finetuned components into it. and yes, we do listen, so wiki and docs are updated regularly - did you check recently?
third, i don't know what you are doing and whats broken for you - best to create an issue and document it.

**Reply by panton41** [reply]
I apologize. I was hungover, awake for 20 hours and spent half the night reinstalling Linux and all the crap it took to make SD.Next work with an AMD GPU, and getting everything else set up, while half drunk. I would not advise that in the future, but no good story begins with "So, I was drinking water."

I'll make a new post and go from there.

**Reply by vladmandic** [reply]
i'll take a closer look tomorrow.

**Comment by TanvirHafiz** [comment]
i downloaded the model and put it in model folder. it picked it up but couldnt use it. whats the proper process to download everthing that i need and run it?

**Reply by vladmandic** [reply]
i dont know what you've downloaded - read what i just wrote above

**Reply by TanvirHafiz** [reply]
i tried that and also downloaded the safetensor file separately. can you give a detail or point me towards a video tutorial that shows how run SD 3.5 in SD next. i havent found any btw

**Reply by vladmandic** [reply]
you tried what i said? that works for thousands of users. show me step-by-step with log. if there are issues, fine, we can work on that, but this is running in circles.

**Reply by TanvirHafiz** [reply]
okay downloading via huggingface through the interface

**Reply by TanvirHafiz** [reply]
21:23:46-907515 ERROR    Diffusers download error: id="stabilityai/stable-diffusion-3.5-large" model access requires
                         login
21:23:46-952519 INFO     Available Models: path="models\Stable-diffusion" items=1 time=0.04
21:23:46-957515 INFO     Diffuser model downloaded: model="stabilityai/stable-diffusion-3.5-large"

i put in the HF token, it says diffuser downloaded but actually nothing was downloaded. also i cant find any way to give some kind og login credential. or am i missing something here

**Reply by vladmandic** [reply]
> start with networks -> models -> reference and select a model.

not download. this.

**Comment by TanvirHafiz** [comment]
yes i did that. so i need to have the model in my folder and then select this? the message

21:23:46-907515 ERROR Diffusers download error: id="stabilityai/stable-diffusion-3.5-large" model access requires
login
21:23:46-952519 INFO Available Models: path="models\Stable-diffusion" items=1 time=0.04
21:23:46-957515 INFO Diffuser model downloaded: model="stabilityai/stable-diffusion-3.5-large"

was in the CLI after i did that. sorry to keep on knocking you!

**Reply by vladmandic** [reply]
so enter your huggingface token to sdnext settings so it can use it.
sdnext cannot bypass model restrictions if they are set by author. and this is covered in wiki in several places.

**Reply by TanvirHafiz** [reply]
firstly the network model downloader do not have any place for entering token. i was able to download models like kandisky 3 using that. but when i tried with SD 3.5 large it asks for password (not token). there is also a place to download model in the models tab where i can in fact enter a huggingface token. even there it asks for pasword.

when i try using the network box i get this

11:55:47-358870 ERROR    Diffusers download error: id="stabilityai/stable-diffusion-3.5-large" model access requires
                         login
11:55:47-365848 ERROR    Reference download: model="stabilityai/stable-diffusion-3.5-large"

and when i tried to download using models tab it says this

11:58:02-059634 INFO     HF login: token="C:\Users\steam\.cache\huggingface\token" Token is valid (permission: write).
Diffusers ?it/s                                0% 0/45 00:00 ? Fetching 45 files
11:58:04-031635 ERROR    Diffusers download error: id="stabilityai/stable-diffusion-3.5-large" model access requires
                         login
11:58:04-039632 INFO     Available Models: path="models\Stable-diffusion" items=1 time=0.00
11:58:04-045634 INFO     Diffuser model downloaded: model="stabilityai/stable-diffusion-3.5-large"

but does nothing. please note that i have entered by HF token key.

**Reply by TanvirHafiz** [reply]
is there any way to login via the interface or athorize it?

**Reply by vladmandic** [reply]
https://github.com/vladmandic/automatic/wiki/Gated
